docs/sources/setup/install/helm/concepts.md

```diff
@@ -15,17 +15,27 @@ keywords:
 
 This section describes the components installed by the Helm Chart.
 
-## Loki read and write
+## 3 methods of deployment
 
-By default Loki will be installed in the scalable mode. This consists of a read and write component. These can be scaled-out independently.
+The loki chart supports three methods of deployment:
+- [Monolithic]({{< relref "./install-monolithic" >}}) 
+- [Simple Scalable]({{< relref "./install-scalable" >}})
+- [Microservice]({{< relref "./install-microservices" >}})
 
-## Dashboards
+By default, the chart installs [Simple Scalable]({{< relref "./install-scalable" >}}). This is the recommended method for most users. To understand the differences between deployment methods, see the [Loki deployment modes]({{< relref "../../../get-started/deployment-modes" >}}) documentation.
+
+## Monitoring Loki
+
+The Loki Helm chart does not deploy self-monitoring by default. Loki clusters can be monitored using the meta monitoring stack, which monitors, the logs, metrics, and traces of the Loki cluster. There are two deployment options for this stack see the installation instructions within [Monitoring]({{< relref "./monitor-and-alert" >}})
+
+{{< admonition type="warning" >}}
+The meta monitoring stack replaces the monitoring section of the Loki helm chart which is now **DEPRECATED**. See the [Monitoring]({{< relref "./monitor-and-alert" >}}) section for more information.
+{{< /admonition >}}
 
-This chart includes dashboards for monitoring Loki. These require the scrape configs defined in the `monitoring.serviceMonitor` and `monitoring.selfMonitoring` sections described below. The dashboards are deployed via a config map which can be mounted on a Grafana instance. The Dashboard requires an installation of the Grafana Agent and the Prometheus operator. The agent is installed with this chart.
 
 ## Canary
 
-This chart installs the [canary]({{< relref "../../../operations/loki-canary" >}}) and its alerts by default. This is another tool to verify the Loki deployment is in a healthy state. It can be disabled with `monitoring.lokiCanary.enabled=false`.
+This chart installs the [canary app]({{< relref "../../../operations/loki-canary" >}}) by default. This is another tool to verify the Loki deployment is in a healthy state. It can be disabled with `lokiCanary.enabled=false`.
```

Comment:
```suggestion
This chart installs the [Loki Canary app]({{< relref "../../../operations/loki-canary" >}}) by default. This is another tool to verify the Loki deployment is in a healthy state. It can be disabled by setting `lokiCanary.enabled=false`.
```